src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2Engine.java

```diff
@@ -58,6 +58,7 @@
     public static final int INDEL_CONTINUATION_QUAL = 10;
     public static final double MAX_ALT_FRACTION_IN_NORMAL = 0.3;
     public static final int MAX_NORMAL_QUAL_SUM = 100;
+    public static final int MIN_PALINDROME_SIZE = 5;
```

Comment:
In offline discussions you said that this could be as low as 4.  Why is it 5 here?

Reply:
These are two different things -- 4 is the number of bases in the artifact, whereas this is a minimum size of non-artifact ITRs required for single-strand hybridization to occur.

Reply:
ok